File: src/library/Publications.java
package library;


public class Publications {
    private int id_pub;
    private String titre_pub;
    private String description_pub;

    public Publications(int id_pub, String titre_pub, String description_pub) {
        this.id_pub = id_pub;
        this. titre_pub= titre_pub;
        this.description_pub= description_pub;


    }



    public int getId_pub() {
        return id_pub;
    }

    public String getTitre_pub() {return titre_pub;}

    public String getDesrciption_pub() {
        return description_pub;
    }


}

File: Gestion_panier/src/Entity/Publications.php
<?php

namespace App\Entity;

use Doctrine\ORM\Mapping as ORM;

/**
 * Publications
 *
 * @ORM\Table(name="publications")
 * @ORM\Entity
 */
class Publications
{
    /**
     * @var int
     *
     * @ORM\Column(name="id_pub", type="integer", nullable=false)
     * @ORM\Id
     * @ORM\GeneratedValue(strategy="IDENTITY")
     */
    private $idPub;

    /**
     * @var string
     *
     * @ORM\Column(name="titre_pub", type="string", length=20, nullable=false)
     */
    private $titrePub;

    /**
     * @var string
     *
     * @ORM\Column(name="description_pub", type="text", length=65535, nullable=false)
     */
    private $descriptionPub;

    /**
     * @var int|null
     *
     * @ORM\Column(name="id_commentaire", type="integer", nullable=true)
     */
    private $idCommentaire;


}
